react-frontend/src/components/StyleComponents/htmlTags.js

```diff
@@ -177,11 +184,49 @@ export const NavBarLink = styled(Link).attrs({
   }
 `;
 
-export const NavBarLinkFirst = styled(NavBarLink).attrs({
-  className: 'navBarLinkFirst',
+export const NavBarLinkResource = styled(Link).attrs({
```

Comment:
Having custom styling on each nav bar element is a red flag css code wise.  in the original pattern there was only 'special' styling on the first element.  The rest shared the same style